db/migrate/20190409231451_create_app_remote_kills.rb

```diff
@@ -0,0 +1,15 @@
+class CreateAppRemoteKills < ActiveRecord::Migration[5.1]
+  def change
+    create_table :app_remote_kills do |t|
+      t.references :app, type: :uuid, null: false, foreign_key: true
+      t.references :split, type: :uuid, null: false, foreign_key: true
+      t.string "reason", null: false
+      t.string "override_to", null: false
+      t.integer "first_bad_version", array: true, null: false
+      t.integer "fixed_version", array: true
```

Comment:
I'm assuming it'll be pretty trivial to go back in and set a fixed version once it is fixed.

Reply:
yeah, that's what the reason is for - you post up another migration for the same split app and reason and it sets fixed_version.